### DISCUSSION TITLE
21.09.25 토 회의록

### DISCUSSION THREAD

**Discussion by ha3158987** [Team-IT-DA/Frontend#162] [회의록]
### 공통 
- 각자 필요한  API(util > API)에 구현해보고 수요일에 다시 얘기하기
- develop pull 받아서 작업하기 
- (시간이 되면) 서버 통신 관련 코드 수정하기

### Daisy 
- 로그인상태(로그인이 되어 있는지, 안되어 있는지) 전역 상태 관리 부분 구현

### Jenny
- util > API > intercepter 등 하위 파일들에 필요한 타입 설정 
- addProduct 쪽 API 붙이기

### Dico
- refactoring 나머지 작업 (myPageTab 상태 관리 관련)
- myPage 서브라우팅 적용해보기 

## 그 외 필요한 사항
- localStorage에서 token을 조회하는 건 util로 관리
- ++ 비용 안드는 클라우드가 있는지 백엔드 분들한테 여쭤보기

**Comment by wjddnjswjd12** [comment]
- notion에 등록된 변수명과 일치하지 않는 부분들이 있음(제니만 한정일듯)
- addProduct쪽 category추가부분도 selectBox로 차후에 구현해둬야함!
- 이미지 추가도 서버분들이랑 api맞춰보고 완료되면 바로 로직 추가하면 될 것 같습니다!
